src/hooks/usePackage.js

```diff
@@ -52,9 +52,9 @@ export const useConfirmPackage = () => {
   const { loading, error, data } = useQuery(CONFIRM_PACKAGE_QUERY, {
     variables: {
       id: packageId,
-      client_name: confirmData.name,
+      'client_name': confirmData.name,
       rut: confirmData.RUT,
-      client_phone: confirmData.phone,
+      'client_phone': confirmData.phone,
```

Comment:
Some keys doesn't have single quotes and others yes. Exists a reason? Please try to maintain a standard

Reply:
I addressed that issue